bin/improver-tests

```diff
@@ -145,17 +145,29 @@ function improver_test_cli {
     fi
 
     for SUBTEST in $CLISUBTEST; do
-        test_dir=$IMPROVER_DIR/tests/improver-$SUBTEST/
-        test_dir="$(echo -e "${test_dir}" | tr -d '[:space:]')"
+        if [[ -f "$SUBTEST" ]]; then
+            # A particular file rather than a subdir name.
+            TEST_TARGET="$SUBTEST"
+        elif [[ -f "$TEST_INPUT_PWD/$SUBTEST" ]]; then
+            # A particular file relative to the original $PWD.
+            TEST_TARGET="$TEST_INPUT_PWD/$SUBTEST"
+        else
+            TEST_TARGET="$IMPROVER_DIR/tests/improver-$SUBTEST/"
```

Comment:
This third case actually captures the following behaviour as well:
```
improver tests cli nbhood/05-mask-square.bats
```

Apparently the extraneous slash on the end doesn't affect the bats runner, and it just runs the one test specified in this way. Not sure if this was intentional, but it's useful.